src/index.html

```diff
@@ -34,8 +34,8 @@
   <main>
 
   </main>
-  <footer>
-
+  <footer class="footer">
```

Comment:
I think better solution is move this footer class in your partial html file. Just add styles from class "footer" to class "footer__container".

Reply:
Done